Products/Zing/datamaps.py

```diff
@@ -313,9 +318,13 @@ def apply_extra_fields(self, om_context, f):
         f.data[ZFact.MetadataKeys.NAME_KEY] = om_context.name
 
         if om_context.is_device:
+            f.data[ZFact.MetadataKeys.ZEN_SCHEMA_TAGS_KEY] = "Device"
```

Comment:
Should we check to make sure it is not also a DeviceComponent? If the latter, we should only add DeviceComponent to the schema-tags, not Device

Reply:
It would only happen if the object was an instance of both Device and DeviceComponent. Is that possible?  If so, yes we should I think.

Reply:
The next commit handles this, to ensure it won't get both schemas from this addition.